src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -167,21 +167,22 @@ public async Task DeleteAsync(string id)
 		public async Task<List<ListingOverviewDTO>> GetAsync(IListingFilterModel? filter = null)
 		{
 			// Get entities from database
+
```

Comment:
Is this whitespace intended?

Reply:
No, I just found it to be more readable that way.